### DISCUSSION TITLE
Different stopwords with a new installation

### DISCUSSION THREAD

**Discussion by adunning** [retorquere/zotero-better-bibtex#2923] [Q&A] [answered]
I have installed Better BibTeX with Zotero on a different computer, using a freshly synchronized library rather than copying over my old Zotero directory. Both the old and new computers are using the default citekey pattern, but the new installation seems to be using a different list of stopwords, resulting in some different citekeys.

For example, the book by Haki Þór Antonsson, *St. Magnús of Orkney: A Scandinavian martyr-cult in context* (2007), <http://doi.org/10.1163/ej.9789004155800.i-272>, previously had the citekey `antonssonMagnusOrkneyScandinavian2007` but is `antonssonStMagnusOrkney2007` on the new installation.

I'm happy to update a few keys in my working files, but I cannot figure out how to make BBT use the same list of stopwords on both installations. Ideally I would like to update the old installation to the latest list. Has anyone encountered a problem similar to this before?

**Comment by retorquere** [comment]
I'll need a debug log from both systems.

**Comment by adunning** [comment, marked as answer]
I was able to fix the problem by opening the Config Editor and resetting the preference `extensions.zotero.translators.better-bibtex.skipWords`.
